Dreamscapes/Ldap/Core/Ldap.php

```diff
@@ -301,10 +301,12 @@ public function add($dn, array $entry)
         return $this;
     }
 
-    /** Alias of self::unbind() */
+    /**
+     * Alias of self::unbind()
+     */
     public function close()
     {
-        return $this->unbind();
+        $this->unbind();
```

Comment:
That is true. The most likely update I can think of would be returning something to indicate the success/failure of the unbinding, and for that the `return` can take care of it for me. I insist.:) Thank you!